doc/cephadm/services/nfs.rst

```diff
@@ -161,6 +161,51 @@ that will tell it to bind to that specific IP.
 Note that in these setups, one should make sure to include ``count: 1`` in the
 nfs placement, as it's only possible for one nfs daemon to bind to the virtual IP.
 
+NFS with HAProxy Protocol Support
+----------------------------------
+
+Cephadm supports deploying NFS in High-Availability mode with additional
+HAProxy protocol support. This works just like High-availability NFS but also
+supports client IP level configuration on NFS Exports.  This feature requires
+NFS-Ganesha ``TODO-version-TODO`` or later.
```

Comment:
v5

https://github.com/nfs-ganesha/nfs-ganesha/wiki/ReleaseNotes_5

Reply:
Hah, thanks for catching that I almost certainly would have forgotten I left that TODO in there!

Reply:
Fixed